docs/_sidebar.json

```diff
@@ -21,6 +21,7 @@
             "tools/slack-cli/guides/troubleshooting-slack-cli-errors",
             "tools/slack-cli/guides/setting-up-ci-cd-with-the-slack-cli",
             "tools/slack-cli/guides/deploying-with-the-slack-cli-and-github-actions",
+            "tools/slack-cli/guides/removing-an-app",
```

Comment:
🔍 issue: I'm not finding this page in the preview, but this seems alright still!